integration_tests/fixtures/forseti/outputs.tf

```diff
@@ -18,10 +18,10 @@ output "bastion_host" {
   value = module.bastion.host
 }
 
-output "forseti-cai-storage-bucket" {
-  description = "Forseti CAI storage bucket"
-  value       = module.forseti.forseti-cai-storage-bucket
-}
+//output "forseti-cai-storage-bucket" {
+//  description = "Forseti CAI storage bucket"
+//  value       = module.forseti.forseti-cai-storage-bucket
+//}
```

Comment:
Should this be removed or un-commented?

Reply:
I am planning on making a quick change to the TF module today to add this as an output of the main module. It is already an output of the server_gcs module, just need to output at one level higher.